src/code/z_scene_table.c

```diff
@@ -350,7 +350,7 @@ void Scene_DrawConfigTempleOfTime(PlayState* play) {
 
     CLOSE_DISPS(play->state.gfxCtx, "../z_scene_table.c", 5145);
 
-    if (gSaveContext.sceneLayer == 5) {
+    if (gSaveContext.sceneLayer == GET_CUTSCENE_LAYER(CS_INDEX_1)) {
```

Comment:
```suggestion
    if (gSaveContext.sceneLayer == 5) {
```

I'd rather just have layers be defined on a per-scene basis than be a function of `CS_INDEX`

Reply:
oh, okay, so it was unnecessary for me to do? yeah, probably easier to read and compare with `entrance_table.h` when it's indexed the same way, from `0`

i was thinking *"hey, there's only two instances of `gSaveContext.sceneLayer` compared with values `[0 .. 3]`"*
- https://github.com/zeldaret/oot/blob/67f0eef7905b695ca17ef7c887ac92590281277c/src/code/z_scene.c#L448
- https://github.com/zeldaret/oot/blob/67f0eef7905b695ca17ef7c887ac92590281277c/src/code/z_scene.c#L457

well, if it makes it more difficult, let's revert these line to the original state

Reply:
The lines in z_scene.c shouldn't be reverted.

The idea is that at some point we will have enumerations for the different scene layers on a per "scene" basis (perhaps making exceptions for Hyrule Market/Castle areas), and use the one that makes the most sense contextually. In `Scene_DrawConfigTempleOfTime`, that 5 might become e.g. `SCENE_LAYER_TEMPLE_OF_TIME_CS_CREDITS_MASTER_SWORD` or something to that effect (perhaps it retains the cutscene index). This is the much simpler solution as it allows you to reference the layer directly.

The only reason these enums haven't been done is we're sort of holding out for someone to rework the assets system to allow more stuff to be committed